packages/http/test/fixtures/index.ts

```diff
@@ -0,0 +1,5 @@
+
+import { VALUES } from './values';
+import { dummyFetch } from './dummyFetch';
+
+export { dummyFetch, VALUES };
```

Comment:
We commonly use the following pattern:

```suggestion
export * from './values';
export * from './dummyFetch';
```